src/Image.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const Image = React.createClass({
+
+  propTypes: {
+    src: React.PropTypes.string,
```

Comment:
That's correct. I meant all of the props that just get forwarded to `img` directly.